compiler/rustc_span/src/hygiene.rs

```diff
@@ -1020,11 +1137,18 @@ pub struct HygieneEncodeContext {
 }
 
 impl HygieneEncodeContext {
+    /// Record the fact that we need to serialize the corresponding `ExpnData`.
+    pub fn schedule_for_encoding(&self, expn: ExpnId) {
```

Comment:
```suggestion
    pub fn schedule_expn_data_for_encoding(&self, expn: ExpnId) {
```
I misinterpreted what this function does, the `ExpnId` itself is encoded immediately, it's `ExpnData` encoding which is delayed.